src/Tools/Source/RunTests/Options.cs

```diff
@@ -171,6 +176,11 @@ bool isOption(string argument, string optionName, out string value)
                     opt.ProcDumpDirectory = value;
                     index++;
                 }
+                else if (comparer.Equals(current, "-procdump"))
+                {
+                    opt.UseProcDump = false;
```

Comment:
Should this be `true`?

Reply:
Oh man ... it's been a banner morning for me today. I should stop everything until I finish my coffee.

Reply:
https://github.com/dotnet/roslyn/pull/31472